care/facility/utils/reports/discharge_summary.py

```diff
@@ -181,7 +179,6 @@ def get_discharge_summary_data(consultation: PatientConsultation):
     return {
         "patient": consultation.patient,
         "samples": samples,
-        "hcx": hcx,
```

Comment:
we should think of a way to allow apps to expose discharge summary template sections and data